definitions/npm/metalsmith_v2.x.x/flow_v0.75.x-/metalsmith_v2.x.x.js

```diff
@@ -71,11 +82,11 @@ declare module 'metalsmith' {
     run(files: FilesMap, plugins: Array<Plugin>, cb: Callback<FilesMap>): void;
     run(files: FilesMap, plugins: Array<Plugin>): FilesMap;
```

Comment:
Do all these methods need versions compatible with `RFile`? If so, then this pattern isn't very efficient.

Reply:
No, only in methods that creates and returns _file_ (i.e. `readFile` and `read`) is guaranteed that the _file_ contains `mode` and `stats` (and no extra properties). `RFile` is de facto subtype of `File`.